python/docs/basics.md

```diff
@@ -1,5 +1,20 @@
 # RMM - the RAPIDS Memory Manager
 
+RMM is a package that enables you to allocate GPU ("device") memory
```

Comment:
It's important to say something about performance, I think.

Reply:
How's this?